_research/research_post.md

```
@@ -0,0 +1,222 @@
+# Adaptive CFD–POD–LSTM Pipeline
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)